### PR TITLE
#313  Create prepare-deploy rule. Recreate nginx container during deploy

### LINKED ISSUE #313: Deploy mech broken
`make deploy` leads to 500 error on prod because of error in `front/build` code shipping

```
# dcp logs app
se-python                | ValueError: The file 'js/index.min.js' could not be found with <django.contrib.staticfiles.storage.ManifestStaticFilesStorage object at 0x7ff92563ce80>.
```

### PR DESCRIPTION
**Pull request by duker33**
Closes #313

### CODE REVIEW

File: docker/Makefile [hunk 1]
```diff
@@ -64,11 +68,22 @@ generate-production-static-data:
...
+	$(dcb) build --no-cache python-dev python-prod
+	$(dcb) push python-dev python-prod
+
 deploy:
 	$(dcp) pull
 	$(dcp) stop
-	$(dcp) rm -f app
-	$(dcp) up -d app
+	$(dcp) rm -f app && $(dcp) up -d app
+	$(dcp) rm -f nginx && $(dcp) up -d nginx
```
Comment (artemiy312) [inline docker/Makefile]:
```bash
$(dcp) rm -f app  nginx && $(dcp) up -d app nginx
# or
# I'd prefer the second approach
$(dcp) rm -f app nginx
$(dcp) up -d app nginx
```
will be better

File: docker/Makefile [hunk 2]
```diff
@@ -64,11 +68,22 @@ generate-production-static-data:
 	$(dcp) exec app python manage.py excel
 	$(dcp) exec app python manage.py price
 
+# drone should do this in working flow.
+# But if drone don't do this for some reasons,
+# you can run this recipe from local machine.
+prepare-deploy:
```
Comment (artemiy312) [inline docker/Makefile]: Seems `deploy-(preprare|push)` will be better. Decide on your own

File: docker/Makefile [hunk 3]
```diff
@@ -64,11 +68,22 @@ generate-production-static-data:
...
+	rm -rf front/build
+	$(dc) stop app && $(dcp) stop app
+	$(dc) rm -f app && $(dcp) rm -f app
```
Comment (artemiy312) [inline docker/Makefile]: 1. Why do you remove an `app` container?
2. You may use here `docker-compose rm -fs ...` to stop and remove container

File: docker/Makefile (hunk 1 above)
Comment (duker33) [reply docker/Makefile]: TODO

**Pull request merged**